src/fswAlgorithms/pointCloudProcessing/cameraTriangulation/cameraTriangulation.h

```diff
@@ -54,12 +54,12 @@ class CameraTriangulation: public SysModel {
 private:
     void readMessages();
     void writeMessages(uint64_t currentSimNanos);
-    Eigen::Vector3d triangulation(std::vector<Eigen::Vector3d> knownLocations,
+    Eigen::Vector3d triangulation(Eigen::MatrixXd knownLocations,
                                   std::vector<Eigen::Vector2d> imagePoints,
                                   const Eigen::Matrix3d& cameraCalibrationInverse,
                                   std::vector<Eigen::Matrix3d> dcmCamera) const;
 
-    std::vector<Eigen::Vector3d> pointCloud{}; //!< [-] point cloud
+    Eigen::MatrixXd pointCloud; //!< [-] point cloud
```

Comment:
:large_orange_diamond: We lost the default initializer syntax with this edit '{}'. Was that desired?

Reply:
I fixed a bunch of these